.github/workflows/unittests.yml

```diff
@@ -22,3 +22,5 @@ jobs:
     uses: N3PDF/workflows/.github/workflows/python-poetry-tests.yml@main
     with:
       python-version: "3.10"
+
+
```

Comment:
This is the effect of missing `pre-commit` installation. It would have removed automatically blank lines at the end of the file.

Reply:
yes sorry for that. I have installed it now